docs/fides/docs/getting_started/docker.md

```diff
@@ -2,47 +2,91 @@
 
 ---
 
-The recommended way to get Fidesctl running is to launch it using the supplied `make` commands. The `make` commands wrap `docker-compose` commands that will spin up each piece of the project.
+The recommended way to get Fidesctl running is via Docker. The following guide will describe how to get things going, step-by-step.
 
 ## System Requirements
 
-1. Install `make` locally (see [Make on Homebrew (Mac)](https://formulae.brew.sh/formula/make), [Make for Windows](http://gnuwin32.sourceforge.net/packages/make.htm), or your preferred installation)
-1. Install `docker` locally (see [Docker Desktop](https://www.docker.com/products/docker-desktop) or your preferred installation; use a recent enough version so that `docker-compose` is also available)
-1. Clone the [Fidesctl repo](https://github.com/ethyca/fides) and `cd` into the `fides` directory
+Docker is the only requirement here. Install `docker` locally (see [Docker Desktop](https://www.docker.com/products/docker-desktop) or your preferred installation). The minimum verified Docker version is `20.10.8`
 
 ## Docker Setup
 
-The following commands should all be run from the top-level `fides` directory (where the Makefile is):
+1. Create a network for your containers to use:
 
-1. `make cli` -> This will spin up the entire project and open a shell within the `fidesctl` container, with the `fidesapi` being accessible. This command will "hang" for a bit, as `fidesctl` will wait for the API to be healthy before launching the shell. Once you see the `fidesctl#` prompt, you know you're ready to go:
+    ```shell
+    docker network create fidesctl-net
+    ```
 
-    ```bash
-    ~/git/fides% make cli
-    Build the images required in the docker-compose file...
-    ...
-    Building fidesapi
-    ...
-    Building fidesctl
-    ...
-    Building docs
-    ...
-    root@1a742083cedf:/fides/fidesctl#
+1. Spin up a Postgres database:
+
+    ```shell
+    docker run --name postgres-docker --net fidesctl-net -d -e "POSTGRES_PASSWORD=postgres" -p 5432:5432 postgres:12
+    ```
+
+1. Start the Fidesctl webserver:
+
+    ```shell
+    docker run --name fidesctl-docker --net fidesctl-net -e "FIDESCTL__API__DATABASE_URL=postgresql+psycopg2://postgres:postgres@postgres-docker:5432/postgres" -p 8080:8080 -d ethyca/fidesctl fidesctl webserver
+    ```
+
+1. Attach a shell to the Fidesctl container:
+
+    ```shell
+    docker exec -it fidesctl-docker /bin/bash
     ```
 
-1. `fidesctl init-db` -> Builds the required images, spins up the database, and runs the initialization scripts:
+1. Init the database:
 
     ```bash
     ~/git/fides% fidesctl init-db
     INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
```

Comment:
Since the majority of this snippet is output, the syntax highlighting threw me.
```suggestion
    ```
    ~/git/fides% fidesctl init-db
    #=>
	INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
```